src/modules/template-documentation/TemplateDocumentation.tsx

```diff
@@ -59,8 +64,8 @@ const TemplateDocumentation = () => {
   );
 
   useEffect(() => {
```

Comment:
You should get error property from documentationProvider hook and check inside useEffect - if error truthy then redirect to /app/templates

Reply:
should it redirect to /app/templates or specific template like: /app/templates/all/:id?

Reply:
/app/templates will be fine